tensorflow/python/feature_column/sequence_feature_column_test.py

```diff
@@ -1553,6 +1553,8 @@ def _custom_fn(input_tensor):
     self.assertEqual(column.shape, (2,))
     self.assertEqual(column.default_value, 3)
     self.assertEqual(column.normalizer_fn(3), 45)
+    with self.assertRaises(ValueError):
```

Comment:
Thanks-- can you use assertRaisesRegex to ensure the error caught is the one expected?

Reply:
Thanks. Updated!